### ISSUE TITLE
Machine Learning Week 2 Quiz 1 (Linear Regression with Multiple Variables) Stanford Coursera

### ISSUE THREAD

**Issue by Adesoji1** [mGalarnyk/datasciencecoursera#42]
Answer | Explanation
-- | --
α=0.3 is an effective choice of learning rate. | We want gradient descent to quickly converge to the minimum, so the current setting of α seems to be good,  X[WRONG]

it is wrong. The learning rate &=0.3 still looks high compared with 0.1. The right answer is or should be; Rather than use the current value of α, it'd be more promising to try a smaller value of α (say α=0.1).

**Comment by AasthaSinha2002** [comment]
Even if learning rate 0.1 is smaller but it isn't the right choice as choosing both higher values and extremely low values can cause the graph to diverge and slow to converge respectively.
<img width="859" alt="Screenshot 2021-08-12 at 12 55 27 AM" src="https://user-images.githubusercontent.com/74906694/129090630-36a6c739-7a69-41b1-975e-0c4c771286cc.png">

**Comment by AasthaSinha2002** [comment]
Even if the learning rate 0.1 is smaller it isn’t the right answer as choosing both higher and extremely low value can cause the divergence of graph and can cause the learning rate to converge slow respectively.
<img width="859" alt="Screenshot 2021-08-12 at 12 55 27 AM" src="https://user-images.githubusercontent.com/74906694/129092112-b18b631e-d94c-4ab7-a324-a115e3476e9f.png">
